sites.json

```diff
@@ -1666,11 +1666,14 @@
     },
 
     {
-        "name": "Discord",
+        "name": "Discord(app)",
```

Comment:
Why `Discord(app)` instead of `Discord`? They don't call it `Discordapp`, usually.

Reply:
Both names are pretty commonly used

Reply:
The website exclusively refers to it as "Discord", as do its various legal pages.